shoop/themes/classic_gray/plugins/__init__.py

```diff
@@ -0,0 +1,16 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+
+from product_highlight import ProductHighlightPlugin
```

Comment:
You should use ``from .product_highlight import ProductHighlightPlugin`` and same goes with the next to imports in this file.